zenodo/modules/openaire/helpers.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Zenodo.
+# Copyright (C) 2017 CERN.
+#
+# Zenodo is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""OpenAIRE related helpers."""
+
+from __future__ import absolute_import, print_function
+
+import hashlib
+
+from flask import current_app
+
+#: ID prefix for OpenAIRE publications.
+OPENAIRE_ID_PREFIX_PUBLICATION = 'od______2659'
```

Comment:
There's two solutions here: either moving these to `zenodo.config`, or adding `zenodo/modules/openaire/{config,ext}.py` and moving [this section](https://github.com/zenodo/zenodo/blob/master/zenodo/config.py#L272-L277) there along with the new stuff...